hack/make-rules/tools.mk

```diff
@@ -0,0 +1,23 @@
+INSTALL_TOOLS += $(TOOLBIN)/kind
+$(TOOLBIN)/kind:
+	GOBIN=$(ABSTOOLBIN) go install sigs.k8s.io/kind@v0.11.1
+
+INSTALL_TOOLS += $(TOOLBIN)/yq
+.PHONY: $(TOOLBIN)/yq
+$(TOOLBIN)/yq:
+	cd $(TOOLS_DIR); ./install_yq.sh
+
+INSTALL_TOOLS += $(TOOLBIN)/kubebuilder
```

Comment:
why do you need kubebuilder here?

Reply:
for kubectl, it is needed for kind target